### PR TITLE
Add Open Source Policy

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Copied and slightly modified the open source policy from 18-F.
See: https://github.com/18F/open-source-policy

## 💭 Motivation and Context

We have new people publishing code to cisagov and they need to know this.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: open-source-policy/practice.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+# Practicing our open source policy #
```
Comment (jsf9k) [inline open-source-policy/practice.md:1]: In `policy.md` titles are capitalized.  Here they are not.  I feel like we should be consistent.

I prefer the style used here.

File: open-source-policy/practice.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+CISA currently has the following CLAs signed:
+
+* None 😒
+
+## How to license CISA repos ##
+
+When creating a repo, we highly recommend that you start from one of our
+maintained skeleton projects.  This will quickly get you setup with the correct
+LICENSE and CONTRIBUTING documents as well as some spiffy tooling to keep your
+project healthy.  See the our [project setup](/project_setup) document for the
```
Comment (jsf9k) [inline open-source-policy/practice.md]: See the our --> See our

File: open-source-policy/policy.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+CISA would like to thank 18-F, Consumer Financial Protection Bureau, Department
+of Defense, and Office of Management and Budget for their work in blazing the
+path for the use of FOSS in the Federal Government.
+
+## Future Changes ##
+
+This policy is a living document. CISA expects to make changes to this policy in
+the future, and we welcome
+[issues](https://github.com/cisagov/development-guide/issues) and pull requests.
```
Comment (jsf9k) [inline open-source-policy/policy.md]: Do you want "pull requests" to be a link as well?

**Review by jsf9k — APPROVED** [review]
Approved, with a few minor comments.  Nice work!

File: open-source-policy/policy.md (hunk 3 above)
Comment (felddy) [reply open-source-policy/policy.md]: Good idea!

**Review by dav3r — APPROVED** [review]
If it's good enough for 18F, it's good enough for me! 📝

**Pull request merged**